src/main.rs

```diff
@@ -0,0 +1,169 @@
+use typedb_client::{
+    concept::{Attribute, Concept, Value},
+    Connection, DatabaseManager, Session,
+    SessionType::{Data, Schema},
+    TransactionType::{Read, Write}
+};
+use futures::{StreamExt};
+use  std::fs;
+use text_io::read;
+
+const TEST_DATABASE: &str = "menuDB";
```

Comment:
This isn't our test database! Let's call it `APPLICATION_DATABASE` or `MENU_DATABASE`.

Reply:
Got it. Done